.github/workflows/cd.yml

```diff
@@ -230,6 +242,41 @@ jobs:
           npm publish --access public
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Build wheels (linux)
+        if: matrix.build.PYPI_PUBLISH == true && matrix.build.NAME.startsWith('linux-')
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.build.TARGET }}
+          args: --manifest-path git-cliff/Cargo.toml --release --out dist
+          sccache: 'true'
+          manylinux: auto
+      - name: Build wheels (macos)
+        if: matrix.build.PYPI_PUBLISH == true && matrix.build.OS.startsWith('macos-')
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.build.TARGET }}
+          args: --manifest-path git-cliff/Cargo.toml --release --out dist
+          sccache: 'true'
+      - name: Build wheels (windows)
+        if: matrix.build.PYPI_PUBLISH == true && matrix.build.OS.startsWith('windows-')
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.build.TARGET }}
+          args: --manifest-path git-cliff/Cargo.toml --release --out dist
+          sccache: 'true'
+      - name: Build wheels (musl)
+        if: matrix.build.PYPI_PUBLISH == true && matrix.build.OS.endsWith('musl-')
```

Comment:
I think this should be `matrix.build.OS.endsWith('musl')` (mind the "-")

Reply:
Good catch!